docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -293,6 +298,79 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+## Sending and Receiving Custom Events
+
+Custom commands are a feature that enables the client application to talk to the server application.
+The communication in the opposite direction is done with the help of custom events.
+
+<a name="receive_custom_events"></a>
+
+1. To listen to the custom events on the client-side:
+
+   1. Define your custom event listener callback function:
+
+      ```javascript
+      function eventReceivedCallback(serverId, event, data) {
+          console.log('Event ' + event + ' received from ' + serverId);
+          console.log('Event data: ' + JSON.stringify(data));
+
+          var status = 0;
+          var data = {'test': 'data'};
+          return tizen.mediacontroller.RequestReply(data, status);
+      }
+      ```
+
+      Event handler function can return the `RequestReply` object which will be send back to the event author.
+      `RequestReply` consists of status code integer and bundle data object.
+
+   2. Set the events listener:
+
+      ```javascript
+      mcClient.setCustomEventListener(eventReceivedCallback);
+      ```
+
+   3. Disable the events listener, when your application is no longer listening to the custom events:
+
+      ```javascript
+      mcClient.unsetCustomEventsListener();
+      ```
+
+<a name="send_custom_events"></a>
+
+2. To send the custom events on the server-side:
+
+   1. Define the name and data of the custom event parameters:
+
+      ```javascript
+      var eventName = 'HelloWorld';
+      var eventData = new tizen.Bundle({'testKey': 'testValue'});
+      ```
+
+   2. Prepare reply callback function:
+
+      ```javascript
+      function eventReplyReceived(data, code) {
+          console.log('client reply code: ' + code);
+          console.log('client reply data: ' + JSON.stringify(data));
+      }
+      ```
+
+   3. Retrieve the recipient client info object:
+
+      ```javascript
+      var recipient = mcServer.getAllClientsInfo()[0];
+      ```
+
+      This example assumes that there is only one client application currently running on the device.
+      Otherwise, iterate all the `MediaControllerClientInfo` objects returned by the `getClientInfo()`
+      method to select the recipient.
+
+   4. Send the event:
+
+     ```javascript
```

Comment:
From line 370 to line 372:
Please add one more spaces in front of each line.

Reply:
In Custom command section, I can see the code blocks are aligned well as below image.

![image](https://user-images.githubusercontent.com/9958374/66806612-f2bfcb80-ef62-11e9-9eaf-fe2b0f30d309.png)

However in Custom events section, this code block is out of align.

![image](https://user-images.githubusercontent.com/9958374/66806731-3581a380-ef63-11e9-91a5-476dbfb5e2fe.png)

Or, is this alignment intended?

![image](https://user-images.githubusercontent.com/9958374/66806831-77aae500-ef63-11e9-80dd-74b217579b73.png)

When I compare the code, we have 5 spaces from line 370 to line 372, not like 6 spaces in Custom command.

Reply:
You are correct, I have incorrectly indented the code fragment in section 2.4. I pushed the update. Please verify.